lib/opensusebasetest.pm

```diff
@@ -927,6 +927,25 @@ sub select_user_serial_terminal {
     select_serial_terminal(0);
 }
 
+=head2 upload_coredumps
+
+ upload_coredumps();
+
+Upload all coredumps to logs
+=cut
+sub upload_coredumps {
+    my $res = script_run("coredumpctl -q", timeout => 10);
```

Comment:
Better run it with `--no-pager` ?

Reply:
ok, but still `  -q --quiet             Do not show info messages and privilege warning`